docs(cli): add shortcuts to header section, created contributors.md (#359)
code-pushup/cli: CONTRIBUTORS.md

## Contributing

> ### Setup

Reviewer: The heading levels should be moved up now that this section is a standalone document (`##` :arrow_right: `#`, `###` :arrow_right: `##`, `####` :arrow_right: `###`).

Author: done

| `scope:shared`      | data models, utility functions, etc. (not specific to core or plugins)       | `scope:shared`                                     |
| `scope:tooling`     | supplementary tooling, e.g. code generation                                  | `scope:shared`                                     |
| `type:app`          | application, e.g. CLI or example web app                                     | `type:feature`, `type:util` or `type:testing-util` |
| `type:feature`      | library with business logic for a specific feature                           | `type:util` or `type:testing-util`                 |
| `type:util`         | general purpose utilities and types intended for reuse                       | `type:util` or `type:testing-util`                 |
| `type:e2e`          | E2E testing                                                                  | `type:app`, `type:feature` or `type:testing-util`  |
| `type:testing-util` | testing utilities                                                            | `type:util`                                        |

> #### Special Targets

Reviewer 2: ```suggestion
#### Special targets
```

The repository includes a couple of common optional targets:

- `perf` - runs micro benchmarks of a project e.g. `nx perf utils` or `nx affected -t perf`

> #### Special Folder

Reviewer 2: ```suggestion
#### Special Folders
```

The repository standards organize reusable code specific to a target in dedicated folders at project root level.
This helps to organize and share target related code.

The following optional folders can be present in a project root;

> - `perf` - micro benchmarks related code
> - `mocks` - testing related code
> - `docs` - docs related files
> - `tooling` - tooling related code

Reviewer 2: ```suggestion
- `perf` - micro benchmarks related code
- `mocks` - test fixtures and utilities specific for a given project
- `docs` - files related to documentation
- `tooling` - tooling related code
```
